hedera-node/src/main/java/com/hedera/services/ledger/properties/TokenRelProperty.java

```diff
@@ -71,37 +70,4 @@ public Function<MerkleTokenRelStatus, Object> getter() {
 			return MerkleTokenRelStatus::isAutomaticAssociation;
 		}
 	},
-	KEY {
```

Comment:
So we would always need a mutable MerkleTokenRelStatus when ever we want to change the keys ?

Reply:
True; the `MapValueListUtils` ensure this.